backend/src/db_managers/base.py

```diff
@@ -73,23 +74,20 @@ async def create(self, obj_data: dict[str, Any], session: AsyncSession = None) -
 
         return obj
 
-    async def update(self, id_: int, update_data: dict[str, Any], session: AsyncSession = None) -> T:
+    async def update(self, id_: UUID, update_data: dict[str, Any], session: AsyncSession = None) -> T:
         """Update an object by its ID."""
         session = session or self.session
         instance = await self.get_or_404(id_, session)
 
         for key, value in update_data.items():
             setattr(instance, key, value)
 
-        try:
-            await session.commit()
-            await session.refresh(instance)
-        except IntegrityError as e:
-            await self._handle_integrity_error(e, session)
+        await session.commit()
+        await session.refresh(instance)
```

Comment:
why?

Reply:
I move processing of the exception to the UserService class in update_user method